2024/ig-exploration.html

```diff
@@ -147,25 +147,34 @@ <h1 id="title"><i class="todo">DRAFT</i> Exploration Interest Group Charter</h1>
 
       <div id="background" class="background">
         <h2>Motivation and Background</h2>
-
         <p>
-          The goal of Exploration Interest Group provides a platform to help W3C community
+          The goal of Exploration Interest Group provides a platform to help the W3C community
           investigating emerging technology trends, analyzing their impacts on the evolution
           of Web technologies, and proposing ways for W3C to collaborate shaping the trends
           for the benefit of the Web users. The ways include organizing Workshops, publishing
           Reports and creating topic oriented Task Forces etc., assisting the Web community
           seeking innovative responses to continually strengthen the innovation of W3C and the
           creativity of Web technologies.
         </p>
+        <p>
+          The Exploration Interest Group also provides a forum for:
+        </p>
+        <ul>
+          <li>the
+          <a href='https://www.w3.org/policies/process/#team'>W3C Team</a> to refine
+          charter drafts;</li>
+          <li>the <a href="https://www.w3.org/community/groups/">community groups</a> to
+          receive feedback on their potential next steps.</li>
+        </ul>
       </div>
 
       <section id="scope" class="scope">
         <h2>Scope</h2>
         <ul>
 <li>Monitor industry and technology trends that might impact the web;</li>
-<li>Monitor responses from the industries, SDOs (including W3C CGs, TPAC Breakout sessions, Workshops, Liaison Reports), and Open Source communities on those trends. Work with W3C Team to invite W3C Liaisons to SDOs or other insightful SDOs experts, reporting back on potential areas of investigations;</li>
-<li>Prioritize the trends and propose organizing Workshops, publishing Reports or creating topic oriented Task Forces, for further elaboration;</li>
-<li>Propose chartering potential topics to the Strategy Funnel at the stage of incubation;</li>
+<li>Monitor responses from the industries, SDOs (including W3C CGs, TPAC Breakout sessions, <a href='https://www.w3.org/events/workshops/'>Workshops</a>, <a href='https://www.w3.org/liaisons/'>Liaison Reports</a>), and Open Source communities on those trends. Work with W3C Team to invite W3C Liaisons to SDOs or other insightful SDOs experts, reporting back on potential areas of investigations;</li>
```

Comment:
```suggestion
<li>Monitor responses from the industries, SDOs (including W3C CGs, TPAC Breakout sessions, <a href='https://www.w3.org/events/workshops/'>Workshops</a>, <a href='https://www.w3.org/liaisons/'>Liaison Reports</a>), and Open Source communities on those trends. When asked by the W3C Team, work with them to invite W3C Liaisons to SDOs or other insightful SDO experts, reporting back on potential areas of investigations;</li>
```

Reply:
Not sure about "When asked by the W3C Team". The IG can certainly propose.

Reply:
It's true that anyone can propose ideas to the Team. Do we need charter language in that case? How about deleting the second sentence? That makes it clearer that the scope of activity of the IG is to monitor the ecosystem, which seems like the true focus here.